protocol/flatbuffers/rpc.fbs

```diff
@@ -181,3 +188,31 @@ table AutoBoneEpochResponse {
     epoch_error: float32;
     adjusted_skeleton_parts: [SkeletonPart];
 }
+
+enum FkToggle: uint8 {
```

Comment:
Are these mutually exclusive settings, or are they all able to be on or off at the same time?

Reply:
They're all independent from eachothers, they could all be on or all off, or half off and half on...

Reply:
```suggestion
enum FkSetting: uint8 {
```

Reply:
How about FkValues then?

Reply:
Wdym

Reply:
So you wanna rename toggles to settings right?
1. Do we rename values to something else as well?
2. 2. Wouldn’t this conflict with FkSettings (that contains both fktoggles and fkvalues)? What would you rename that?
Tbh I’d rather just keep it as it is rn, as it also corresponds to how it’s named in the server…

Reply:
I'll take a closer look at this soon, but no it doesn't matter what its called in the server. This naming is unclear and very confusing.